src/Automation/Automation.Test/ScenarioTests/UpdateDynamicGroupPrePostTests.ps1

```diff
@@ -0,0 +1,221 @@
+# ----------------------------------------------------------------------------------
+#
+# Copyright Microsoft Corporation
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# http://www.apache.org/licenses/LICENSE-2.0
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ----------------------------------------------------------------------------------
+
+$rg = "mms-wcus"
+$aa = "JemalOMSAutomation"
+$azureVMIdsW = @(
+        "/subscriptions/cd45f23b-b832-4fa4-a434-1bf7e6f14a5a/resourceGroups/mms-wcus/providers/Microsoft.Compute/virtualMachines/JemalCmdlet1",
```

Comment:
@cormacpayne    the test needs prepacked Vms that has all the requirements  for the feature to run successfully. The hardcoded in the test is the azure resource id for these vms. It will be little difficult to prepare these Vms with all the requirements in the test it self.

if the user doesn't have permission on the resources and would like to re record the test, the user can change the resources to the one he has permission and record the test.   It is not that difficult to change the resource ids  and record the test

Reply:
@jemex for tests that require additional setup, if there's no way around hardcoding resource ids, then you should leave a comment above this section outlining what a user will need to do (either in the portal or using commands) in order to get this test working correctly